spec/requests/api/v1/users/create_spec.rb

```diff
@@ -0,0 +1,122 @@
+require 'rails_helper'
+
+describe 'POST api/v1/users/', type: :request do
+  let(:user)            { User.last }
+  let(:failed_response) { 422 }
```

Comment:
I think that you can get rid of these variables an just use the http status codes. Check out the codes [here](https://guides.rubyonrails.org/layouts_and_rendering.html#the-status-option) 

Then use it like this:
`expect(response).to have_http_status(:success)`